essentials-framework/Essentials Devices Common/Essentials Devices Common/VideoCodec/IConvertiblePreset.cs

```diff
@@ -0,0 +1,9 @@
+﻿using PepperDash.Essentials.Core.Presets;
+
+namespace PepperDash.Essentials.Devices.Common.VideoCodec
+{
+    public interface IConvertiblePreset
+    {
+        PresetBase ReturnConvertedCodecPreset();
```

Comment:
Can we change this name to something like ` ConvertCodecPreset`?

Reply:
Makes sense.